crates/stdarch-test/src/lib.rs

```diff
@@ -190,7 +190,7 @@ pub fn assert(shim_addr: usize, fnname: &str, expected: &str) {
     }
 }
 
-pub fn assert_skip_test_ok(name: &str) {
+pub fn assert_skip_test_ok(_name: &str) {
```

Comment:
This seems to be a mistake.

Reply:
oops, thanks!  I think I was fixing the panic one (it doesn't work in 2018 edition), and somehow this got introduced.

Reply:
I don't think there's anything stopping us from upgrading to 2021 edition so we can do the same for `panic!`. Feel free to do it in this PR if you want.

Reply:
@Amanieu thx, done